site/documentation/content/api/device-connection/index.md

```diff
@@ -107,6 +107,176 @@ Implementors of this API may return a *404* status code in order to indicate tha
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+## Remove command-handling protocol adapter instance for device
+
+Clients use this command to *remove* the information, which protocol adapter instance is currently handling command & control messages for the given device. The mapping information is only removed, if the currently associated adapter instance matches the one given in the request.
+
+This API doesn't mandate checks on the validity of the given device in order not to introduce a dependency on the *Device Registration API*. However, implementations of this API may choose to perform such checks or impose a restriction on the overall amount of data that can be stored per tenant in order to protect against malicious requests.
+
+**Message Flow**
+
+
+{{< figure src="remove_cmd_handling_adapter_instance.svg" title="Client removes the command-handling protocol adapter instance information for device" alt="A client sends a request message for removing the adapter instance mapping and receives a response containing a confirmation" >}}
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on a message to remove the mapping information regarding the command-handling protocol adapter instance for a device in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name                  | Mandatory | Location                 | AMQP Type | Description |
+| :-------------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *subject*             | yes       | *properties*             | *string*  | MUST be set to `remove-cmd-handling-adapter-instance`. |
+| *adapter_instance_id* | yes       | *application-properties* | *string*  | The identifier of the protocol adapter instance to remove the mapping information for. Only if this adapter instance is currently associated with the device or gateway identified by the *device_id* property, the mapping entry will be removed. |
+
+The body of the message SHOULD be empty and will be ignored if it is not.
+
+**Response Message Format**
+
+A response to a *set command-handling adapter instance for device* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message's *status* property may contain the following codes:
+
+| Code  | Description |
+| :---- | :---------- |
+| *204* | OK, the adapter instance mapping information for the device has been removed. |
+| *400* | Bad Request, the command-handling adapter adapter instance information has not been updated due to there being no such mapping entry or due to missing data in the request. |
```

Comment:
I do not quite understand what
> has not been updated due to there being no such mapping entry 

means. The next paragraph describes that a `412` could be returned if the *device_id* is currently not mapped to the *adapter_instance_id* from the request. Is this the same case?

Reply:
Yes, I have rephrased the sentence above.